### PR TITLE
Fix go formatting issue, fix typo

### PR DESCRIPTION
**Pull request by deining**
This PR fixes a typo I spotted in the project.
It also resolves go formatting issues.

### CODE REVIEW

File: encode.go [hunk 1]
```diff
@@ -56,11 +56,11 @@ func (e *MarshalerError) Error() string {
 //
 // Examples of struct field tags and their meanings:
 //
-//   // Field appears in INI as key "myName".
-//   Field int `ini:"myName"`
+//	// Field appears in INI as key "myName".
+//	Field int `ini:"myName"`
```
Comment (subpop) [inline encode.go:60]: This appears to be an IDE whitespace change that isn't directly related to this PR. I'd suggest dropping this change or moving it to a separate PR if you think 1 tab should be used instead of 3 spaces.

**Review by subpop — CHANGES_REQUESTED** [review]
Thanks for catching that! There's one small change I'd like you to consider. It looks like some auto-space formatting changes got caught up in your commit.

File: encode.go (hunk 1 above)
Comment (deining) [reply encode.go:60]: > This appears to be an IDE whitespace change

No. Have a look at the [go report card](https://goreportcard.com/report/github.com/subpop/go-ini) for your project. It complains about `gofmt` issue in `encode.go`. This issue is addressed here.

> that isn't directly related to this PR.

Yes, I intermingled two different issues here, sorry.

> I'd suggest dropping this change or moving it to a separate PR

Please advice whether you are willing to accept both changes in one single PR or whether I should split this PR up into two PRs. Thanks.

Comment (subpop) [reply encode.go:60]: If you split the change into two commits, one PR is fine.

Comment (deining) [reply encode.go:60]: > If you split the change into two commits, one PR is fine.

Done.

**Comment by deining** [comment]
This PR should be ready for merging now.

**Review by subpop — APPROVED** [review]
